### PR TITLE
Level loader

### PR DESCRIPTION
**Pull request by charafau**
Added simple level loader from json
Reformatted levels
Added camera
Added car sprite
Changed color to green
Added loading paths
Added png tiles for grass and road

### CODE REVIEW

File: rapid-router/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="colorPrimary">#3F51B5</color>
-    <color name="colorPrimaryDark">#303F9F</color>
+    <color name="colorPrimary">#4CAF50</color>
```
[the remark this answers is not in the document]
Comment (charafau) [reply rapid-router/src/main/res/values/colors.xml]: Sure, I will also add loading road blocks soon

File: rapid-router/src/main/java/com/codeforlife/rapidrouter/utils/LevelReader.kt [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        val gson = Gson()
+
+        val parser = JsonParser()
+        val jsonObject = parser.parse(text).asJsonObject
+
+        val origin = jsonObject.get("origin")
+        val path = jsonObject.getAsJsonArray("path")
+
+        val paths: List<PathElement> = path.map { jsonElement ->
+            PathElement(Point(jsonElement["coordinate"][0].asInt, jsonElement["coordinate"][1].asInt),
```
[the remark this answers is not in the document]
Comment (charafau) [reply rapid-router/src/main/java/com/codeforlife/rapidrouter/utils/LevelReader.kt]: the main reason for me to change that is to create Point class and.. Gson cannot to that to my knowledge.

File: rapid-router/src/main/res/values/colors.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (JacqMT) [reply rapid-router/src/main/res/values/colors.xml]: The MR is already pretty big. Might be worth branching from this one and having it as a separate review :)

File: rapid-router/src/main/java/com/codeforlife/rapidrouter/utils/LevelReader.kt (hunk 2 above)
[the remark this answers is not in the document]
Comment (JacqMT) [reply rapid-router/src/main/java/com/codeforlife/rapidrouter/utils/LevelReader.kt]: Okay. I'm not sure I fully understand what you mean but I can catch up with you on Monday :)

**Comment by JacqMT** [comment]
Think you need to merge from master but I think the build should pass then :)

**Pull request merged**